editor/edit-post/header/plugins/index.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * External dependencies
+ */
+import { map, isEmpty } from 'lodash';
+import { connect } from 'react-redux';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { NavigableMenu, withInstanceId, IconButton } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { getEllipsisMenuItems } from '../../../api/ellipsis-menu';
+
+/**
+ * Renders a list of plugins that will activate different UI elements.
+ *
+ * @param   {Object} props The component props.
+ * @returns {Object} The rendered list of menu items.
+ */
+function Plugins( props ) {
+	const ellipsisMenuItems = getEllipsisMenuItems();
+
+	if ( isEmpty( ellipsisMenuItems ) ) {
+		return null;
+	}
+
+	const {
+		instanceId,
+	} = props;
+
+	/**
+	 * Handles the user clicking on one of the plugins in the menu
+	 *
+	 * Does nothing currently, but should be used to trigger the plugins sidebar
+	 *
+	 * @param {string} pluginId The plugin id.
+	 * @returns {void}
+	 */
+	function onSelect( pluginId ) {
+		props.onSelect();
+		ellipsisMenuItems[ pluginId ].callback();
+	}
+
+	const plugins = map( ellipsisMenuItems, ( menuItem ) => {
+		return {
+			value: menuItem.name,
+			label: menuItem.title,
+			icon: menuItem.icon,
+		};
+	} );
+
+	const labelId = `components-choice-menu-plugins-label-${ instanceId }`;
+
+	return [
+		<div
+			key="plugins-separator"
+			className="editor-ellipsis-menu__separator" />,
```

Comment:
Formatting for multi-line elements should be:

```js
<div
  key="…"
  className="…"
/>,
```

Reply:
Might be a good time to make a proper `MenuItemsSeparator` or at least `EllipsisMenuSeparator` and refactor the divs in `editor/edit-post/header/ellipsis-menu`.